{ 'cebit_muscle': { 'head': [ { 'duration': 1.512,
                                'name': u'cebit_head_right',
                                'positions': [-1.1307, 0.0126],
                                'starttime': 4.47},
                              { 'duration': 1.512,
                                'name': u'cebit_head_left',
                                'positions': [1.1296, 0.0127],
                                'starttime': 7.022},
                              { 'duration': 1.064,
                                'name': u'ready_state',
                                'positions': [-0.0, 0.0022],
                                'starttime': 9.448}],
                    'left_arm': [ { 'duration': 2.472,
                                    'name': u'cebit_muscle',
                                    'positions': [ -1.7594,
                                                   -1.1309,
                                                   -0.9998,
                                                   -0.1886,
                                                   -0.79,
                                                   -0.25,
                                                   0.82],
                                    'starttime': 1.966},
                                  { 'duration': 0.6820000000000004,
                                    'name': u'cebit_muscle_2',
                                    'positions': [ -2.0013,
                                                   -1.1301,
                                                   -0.9998,
                                                   -0.1901,
                                                   -0.79,
                                                   -0.25,
                                                   0.82],
                                    'starttime': 10.779},
                                  { 'duration': 0.552,
                                    'name': u'cebit_muscle',
                                    'positions': [ -1.7594,
                                                   -1.1309,
                                                   -0.9998,
                                                   -0.1886,
                                                   -0.79,
                                                   -0.25,
                                                   0.82],
                                    'starttime': 11.461},
                                  { 'duration': 0.616,
                                    'name': u'cebit_muscle_2',
                                    'positions': [ -2.0013,
                                                   -1.1301,
                                                   -0.9998,
                                                   -0.1901,
                                                   -0.79,
                                                   -0.25,
                                                   0.82],
                                    'starttime': 12.475},
                                  { 'duration': 0.552,
                                    'name': u'cebit_muscle',
                                    'positions': [ -1.7594,
                                                   -1.1309,
                                                   -0.9998,
                                                   -0.1886,
                                                   -0.79,
                                                   -0.25,
                                                   0.82],
                                    'starttime': 13.091},
                                  { 'duration': 1.954,
                                    'name': u'ready_state',
                                    'positions': [ -1.5135,
                                                   0.7532,
                                                   -0.2755,
                                                   0.0212,
                                                   0.0208,
                                                   -0.2499,
                                                   0.8168],
                                    'starttime': 14.019}],
                    'left_leg': [ { 'duration': 2.024,
                                    'name': u'cebit_muscle',
                                    'positions': [ 0.0219,
                                                   -0.0868,
                                                   -0.6775,
                                                   1.2053,
                                                   0.6213,
                                                   -0.0894],
                                    'starttime': 0.0},
                                  { 'duration': 1.954,
                                    'name': u'ready_state',
                                    'positions': [ 0.0113,
                                                   -0.0746,
                                                   -0.7899,
                                                   1.1893,
                                                   0.6074,
                                                   -0.0704],
                                    'starttime': 15.998}],
                    'right_arm': [ { 'duration': 2.472,
                                     'name': u'cebit_muscle',
                                     'positions': [ 1.7594,
                                                    1.1309,
                                                    0.9998,
                                                    0.1886,
                                                    0.79,
                                                    0.25,
                                                    -0.82],
                                     'starttime': 1.966},
                                   { 'duration': 0.6820000000000004,
                                     'name': u'cebit_muscle_2',
                                     'positions': [ 2.0013,
                                                    1.1301,
                                                    0.9998,
                                                    0.1901,
                                                    0.79,
                                                    0.25,
                                                    -0.82],
                                     'starttime': 10.779},
                                   { 'duration': 0.552,
                                     'name': u'cebit_muscle',
                                     'positions': [ 1.7594,
                                                    1.1309,
                                                    0.9998,
                                                    0.1886,
                                                    0.79,
                                                    0.25,
                                                    -0.82],
                                     'starttime': 11.461},
                                   { 'duration': 0.616,
                                     'name': u'cebit_muscle_2',
                                     'positions': [ 2.0013,
                                                    1.1301,
                                                    0.9998,
                                                    0.1901,
                                                    0.79,
                                                    0.25,
                                                    -0.82],
                                     'starttime': 12.475},
                                   { 'duration': 0.552,
                                     'name': u'cebit_muscle',
                                     'positions': [ 1.7594,
                                                    1.1309,
                                                    0.9998,
                                                    0.1886,
                                                    0.79,
                                                    0.25,
                                                    -0.82],
                                     'starttime': 13.091},
                                   { 'duration': 1.954,
                                     'name': u'ready_state',
                                     'positions': [ 1.5135,
                                                    -0.7532,
                                                    0.2755,
                                                    -0.0212,
                                                    -0.0208,
                                                    0.2499,
                                                    -0.8168],
                                     'starttime': 14.006}],
                    'right_leg': [ { 'duration': 2.024,
                                     'name': u'cebit_muscle',
                                     'positions': [ -0.0219,
                                                    0.0868,
                                                    0.6775,
                                                    -1.2053,
                                                    -0.6213,
                                                    0.0894],
                                     'starttime': 0.0},
                                   { 'duration': 1.954,
                                     'name': u'ready_state',
                                     'positions': [ -0.0113,
                                                    0.0746,
                                                    0.7899,
                                                    -1.1893,
                                                    -0.6074,
                                                    0.0704],
                                     'starttime': 15.998}],
                    'torso': []}}
